nexus/db-model/src/region_snapshot_replacement.rs

```diff
@@ -80,8 +82,13 @@ impl std::str::FromStr for RegionSnapshotReplacementState {
 ///          v                        ---
 ///                                   ---
 ///       Running                     |
-///                                   | set in region snapshot replacement
-///          |                        | finish background task
+///                                   |
+///          |                        |
+///          v                        |
+///                                   | responsibility of region snapshot
+///     Completing                    | replacement finish saga
```

Comment:
nit: there's another version of this diagram (in the saga comments, I think) that has a back-edge from completing to running, though I think this is only if it unwinds, and I'm not sure if you're including those edges here

Reply:
not a nit, that's important! added in 2cd788126